python/kserve/test/test_metrics.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2021 The KServe Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# coding: utf-8
+
+import unittest
```

Comment:
KServe primarily use `pytest` for the unit testing, so let's be consistent with the other tests.

Reply:
okay I actually took this from one of the other tests in the file

Reply:
maybe it was one randomly using unittest

Reply:
I updated it to remove that use case of unittest but I added a mock, seems safer than actually editing an env var.